### ISSUE TITLE
Make tooltip or whatever with Credentials

### ISSUE THREAD

**Issue by cozyfestiv** [James-Lycett/workplace-wellness#27]
Make some sort of display for the login credentials for both Admin and User 

Maybe it could fill those in for you? Is that a thing? Like click here for the user login and then it just fills in the fields for you?

**Comment by James-Lycett** [comment]
yeah maybe a couple little buttons "admin user demo sign in" or something or even just some hyperlinked text "Just here to check out the site? Sign in to our demo [user](https://placekitten.com/200/300) account or demo [admin](https://placekitten.com/100/100) account
